roles/splunk_cluster_master/tasks/configure_indexes.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Add stanza for the index specified
+  ini_file:
+    dest: "{{ splunk.home }}/etc/master-apps/_cluster/local/indexes.conf"
+    section: "{{ index.indexName }}"
+    option: "{{ item.key }}"
+    value: "{{ item.value }}"
+  with_items:
+    - { key: "remotePath", value: "volume:{{ index.remoteName }}/$_index_name" }
+    - { key: "repFactor", value: "auto" }
+
+- name: Add path for the remote specified
+  ini_file:
+    dest: "{{ splunk.home }}/etc/master-apps/_cluster/local/indexes.conf"
+    section: "volume:{{ index.remoteName }}"
+    option: "{{ item.key }}"
+    value:  "{{ item.value }}"
+  with_items:
+    - { key: "storageType", value: "remote" }
+    - { key: "path", value: "{{ index.scheme }}://{{ index.remoteLocation }}" }
+
+- name: Write AWS credentials for the remote specified
+  ini_file:
+    dest: "{{ splunk.home }}/etc/master-apps/_cluster/local/indexes.conf"
+    section: "volume:{{ index.remoteName }}"
+    option: "{{ item.key }}"
+    value: "{{ item.value }}"
+  with_items:
+    - { key: "remote.s3.access_key", value: "{{ index.s3.access_key }}" }
+    - { key: "remote.s3.secret_key", value: "{{ index.s3.secret_key }}" }
```

Comment:
Another thing I just thought of here: when this runs, do the AWS credentials get printed out to the container logs? If so, we should add loop-control around this section so that we don't print plaintext credentials.